src/style.css

```diff
@@ -1,3 +1,77 @@
-#test {
-  color: red;
+* {
```

Comment:
- [ ] OPTIONAL: You manipulate your margins a little, so it appears centered on the screen.